app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -119,22 +120,12 @@ $after-button-padding-left: govuk-spacing(5);
 // Top level navigation and search.
 .gem-c-layout-super-navigation-header__navigation-items,
 .gem-c-layout-super-navigation-header__search-items {
-  background: govuk-colour("light-grey");
   display: block;
   list-style: none;
-  margin: 0 (0 - govuk-spacing(3));
   padding: 0;
-
-  @include govuk-media-query($from: "tablet") {
-    margin: 0 (0 - govuk-spacing(6));
-  }
-
-  @include govuk-media-query($from: "desktop") {
-    background: none;
-    float: left;
-    padding: 0;
-    margin: 0;
-  }
+  background: none;
```

Comment:
Could I check what this style is doing? I'm not seeing a cascading background style from anywhere that we need to override but I might be missing something.